### PR TITLE
create and style login screen, produce token at user login

### PR DESCRIPTION
**Pull request by juliabrowne**

### CODE REVIEW

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Please squash your commits and force push. Thanks!

**Comment by juliabrowne** [comment]
Done! :)

File: js/components/LoginForm/styles.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    shadowColor: '#B5ADAD',
+    shadowOffset: { height: 1, width: 1 },
+    width: width / 1.1,
+    padding: 12
+  },
+  passwordInput: {
+    justifyContent: 'flex-start',
+    borderWidth: 1,
+    borderRadius: 3,
+    borderColor: '#FFFF',
```
Comment (10thfloor) [inline js/components/LoginForm/styles.js]: Please extract all reusable CSS values to the theme CSS file.

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Good work on this feature. Please make a second pass and ensure all CSS values are properly extracted into the theme, where appropriate.

**Comment by juliabrowne** [comment]
done :) 
sorry about the force push!

**Review by 10thfloor — APPROVED** [review]

File: js/screens/UserLogin/styles.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    height,
+    width,
+    justifyContent: 'space-between'
+  },
+  background: {
+    width,
+    height
+  },
+  logoContainer: {
+    ...globalStyles.loginLogoContainer
```
Comment (10thfloor) [inline js/screens/UserLogin/styles.js]: These styles (the ones specific to the login page) are not global, and don't belong in the global css. If you're unsure about which styles are global and which are not,  and how to differentiate, let me know!

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: js/screens/UserLogin/styles.js (hunk 2 above)
Comment (juliabrowne) [reply js/screens/UserLogin/styles.js]: done!

File: js/screens/ActivityDetails/ActivityDetailsContainer.js [hunk 3]
```diff
@@ -13,9 +13,12 @@ export default class ActivityContainer extends Component {
     headerStyle: {
       ...globalStyles.navigationHeader
     }
-  };
-  toggleVisibility = () => {
-    this.setState({ isVisible: !this.state.isVisible });
+  //   titleStyle: {
```
Comment (siddharthparmar7) [inline js/screens/ActivityDetails/ActivityDetailsContainer.js:16]: remove these comments if you don't need this code.

File: js/config/styles.js [hunk 4]
```diff
@@ -7,6 +8,25 @@ const globalStyles = {
...
+    fontSize: 40,
+    fontFamily: 'Quicksand'
+  },
+  logoImageLoginForgotPassword: {
+    height: height / 5,
+    width: width / 4,
+    alignSelf: 'center',
+    resizeMode: 'center',
+    marginTop: 50,
+    marginBottom: -15
```
Comment (siddharthparmar7) [inline js/config/styles.js]: we never assign negative margin and any huge margins like 50. Please change this and use flexbox to your advantage.

File: js/components/LoginForm/LoginForm.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import { View, Text, TextInput, TouchableOpacity } from 'react-native'
+import getAPI from '../../config/api'
+import { Form, Field } from 'react-final-form'
+import styles from './styles'
+export default class LoginForm extends Component {
+  constructor(props) {
```
Comment (siddharthparmar7) [inline js/components/LoginForm/LoginForm.js]: if you don't need the constructor, remove it.

File: js/components/LoginForm/LoginForm.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    super(props)
+  }
+  callingAPI = async values => {
+    const res = await getAPI('/auth/login', {
+      method: 'POST',
+      body: JSON.stringify(values)
+    })
+    console.log(res)
```
Comment (siddharthparmar7) [inline js/components/LoginForm/LoginForm.js]: remove all the console logs.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the required changes. Thank you.

File: js/components/LoginForm/LoginForm.js (hunk 6 above)
Comment (juliabrowne) [reply js/components/LoginForm/LoginForm.js]: done!

File: js/components/LoginForm/LoginForm.js (hunk 5 above)
Comment (juliabrowne) [reply js/components/LoginForm/LoginForm.js]: done!

File: js/config/styles.js (hunk 4 above)
Comment (juliabrowne) [reply js/config/styles.js]: fixed!

File: js/screens/ActivityDetails/ActivityDetailsContainer.js (hunk 3 above)
Comment (juliabrowne) [reply js/screens/ActivityDetails/ActivityDetailsContainer.js:16]: einer's code -- just commented it out so it wouldn't break my code.

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**